packages/cdk8s/package.json

```diff
@@ -32,11 +32,10 @@
   "devDependencies": {
     "@types/follow-redirects": "^1.13.0",
     "@types/jest": "^26.0.20",
-    "@types/json-stable-stringify": "^1.0.32",
-    "@types/node": "^10.17.52",
+    "@types/node": "^10.17.0",
     "@types/yaml": "^1.9.6",
-    "@typescript-eslint/eslint-plugin": "^4.15.0",
```

Comment:
Why the downgrades here?

Reply:
That's the projen synth output. We need to fix in projen if we want to keep this file untampered

Reply:
Ok, can you just disable them from the auto upgrade job for now?